x/superfluid/keeper/intermediary_account_test.go

```diff
@@ -26,6 +26,19 @@ func (suite *KeeperTestSuite) TestIntermediaryAccountCreation() {
 			2,
 			[]superfluidDelegation{{0, 0, 0, 1000000}, {1, 1, 0, 1000000}},
 		},
+		// Can create intermediary account with unbonded, unbonding validators
+		{
+			"test intermediary account with unbonded validator",
+			[]stakingtypes.BondStatus{stakingtypes.Bonded, stakingtypes.Unbonded},
+			2,
+			[]superfluidDelegation{{0, 0, 0, 1000000}, {1, 1, 0, 1000000}},
```

Comment:
Is there a reason we allow intermediary account creation with unbonded validators? Seems odd to me, but I might be missing something.

Reply:
This is just an aside, no change required on this PR in regards to this!